### PR TITLE
feat(ui-kit select): Add top placeholder for Select component [backwards compitable]

### PR DESCRIPTION
**Pull request by IlyaIzr**
Placeholder property is now working for select but previous versions should still work 
image
image
```jsx
<Select 
  label="Animated placeholder" 
  items={items}
  showTopPlaceholder placeholder='i`m placeholder'
>
</Select>
```

### CODE REVIEW

File: packages/ui-kit/src/Select/components/SelectButton/index.tsx [hunk 1]
```diff
@@ -59,6 +63,7 @@ export const SelectButton = React.forwardRef(
       [style.error]: error,
       [style.inline]: type === 'inline',
     });
+    console.log('%c⧭ showTopPlaceholder игеещт', 'color: #00e600', showTopPlaceholder, placeholder);
```
Comment (Akiyamka) [inline packages/ui-kit/src/Select/components/SelectButton/index.tsx:66]: console.log

File: packages/ui-kit/src/Select/components/SelectButton/index.tsx [hunk 2]
```diff
@@ -13,12 +13,14 @@ export interface SelectButtonClasses {
...
   placeholder?: string;
   error?: string;
+  topPlaceholder?: string;
 }
 
 export interface SelectButtonProps {
   label?: string | React.ReactChild | React.ReactChild[];
   className?: string;
   placeholder?: string;
+  showTopPlaceholder?: boolean;
```
Comment (Akiyamka) [inline packages/ui-kit/src/Select/components/SelectButton/index.tsx:23]: showPlaceholderOnTop

File: packages/ui-kit/src/Select/components/SelectButton/index.tsx [hunk 3]
```diff
@@ -68,22 +73,51 @@ export const SelectButton = React.forwardRef(
...
-        placeholderContent = <span className={style.textContent}>{String(item)}</span>;
+
+    const getPlaceholderAndItem = (): ReactNode => {
+      // Several options here
+      // placeholder space - can be occupied with value, with placeholder(multiple options), or Select children
+      // in case item is selected(presented) - top placeholder should be shown in addition
+      // to the item (on top)
+
+      let itemContent: ReactNode;
+      if (item) {
```
Comment (Akiyamka) [inline packages/ui-kit/src/Select/components/SelectButton/index.tsx:84]: Hmm "item"? What is it? I think if we give name for that "something" this code will be much more simpler for understanding

File: packages/ui-kit/src/Select/components/SelectButton/index.tsx [hunk 4]
```diff
@@ -68,22 +73,51 @@ export const SelectButton = React.forwardRef(
...
+        if (multiselect === MULTISELECT_TYPE_AGGREGATE && item && typeof item === 'string') {
```
Comment (Akiyamka) [inline packages/ui-kit/src/Select/components/SelectButton/index.tsx:85]: It not very obvious that empty string in JS falsy value, I suggest more clear way
`multiselect === MULTISELECT_TYPE_AGGREGATE && typeof item === 'string' && item.length`

**Comment by IlyaIzr** [comment]
This solution was implemented over Selecet refactor here https://github.com/konturio/ui/pull/37
So i'm closing this one

**Pull request closed without merging**